Xamarin.Forms.Core/DecorableTextElement.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	[Flags]
+	[TypeConverter(typeof(TextDecorationConverter))]
+	public enum TextDecorations
+	{
+		None = 0,
+		Underline = 1 << 0,
+		Strikethrough = 1 << 1,
```

Comment:
shouldn't it be `StrikeThrough` ?

Reply:
I actually originally had `StrikeThrough` but this was changed after my initial conversation with @jassmith about this PR -- the corresponding APIs on iOS and UWP refer to this as `Strikethrough`, only Android uses `StrikeThrough`

Reply:
whatever, the TypeConverter isn't case sensitive. 👍